Source/OrchestratorMessages.txt

```diff
@@ -78,29 +78,51 @@
+500(F) : "Mothership: Could not create pthread %s. Exiting."
+501(F) : "Mothership: Could not join to pthread %s. Exiting."
+502(W) : "Mothership: Received a message defining an application with name '%s', but the Mothership's application database already holds an application named '%s' that is fully defined (with state '%s'). Recall it first before defining another application with that name."
+503(E) : "Mothership: Error loading supervisor for application '%s': '%s'."
+504(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected non-empty string in field %s. Ignoring message."
+505(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected unsigned in field %s. Ignoring message."
+506(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected a vector of P_Pkt_t values in field %s. Ignoring message."
+507(E) : "Mothership: Thread '%s' received MPI message with key '0x%s', which it did not know how to decode. Ignoring message."
+508(E) : "Mothership: Error opening file '%s' for dumping: %s."
+509(E) : "Instrumentation: %s"
+510(I) : "Log from thread %s: "
+511(W) : "Mothership: Unknown core for thread '0x%s' in '%s' packet. Ignoring."
+512(W) : "Mothership: Unknown application for core '0x%s' in '%s' packet. Ignoring."
+513(W) : "Mothership: Received an unexpected '%s' packet for an application in state '%s'. Ignoring."
+514(W) : "Mothership: Received a command-and-control packet with unknown opcode '0x%s'. Ignoring."
+515(E) : "Mothership: A message for the supervisor for task '%s' contained a packet that could not be handled correctly. Possible causes: 1) The pin field in the packet was set out of bounds, 2) The Supervisor failed to initialise correctly (either the SupervisorInit is broken or was not called), 3) Mark has done something terrible."
+516(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected a vector of uint32_t-P_Pkt_t pairs in field %s. Ignoring message."
+517(W) : "Mothership: Received a packet from an application in error - the isMothership bit was not set in the software address field for the packet. The software address of this packet is '0x%s'. Ignoring packet."
+518(W) : "Mothership: Received a packet for an external device - external support is not yet implemented. The software address of this packet is '%s'. Ignoring packet."
+519(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected char in field %s. Ignoring message."
+520(W) : "Mothership: Invalid task ID '%s' in packet received from the compute fabric. Ignoring packet."
+521(I) : "Debug byte received from thread '0x%s' with content '%s'."
+522(E) : "Mothership: Error decoding MPI message with key '0x%s': Expected a vector of uint32_t values in field %s. Ignoring message."
+523(E) : "Mothership: Supervisor for application '%s' failed to initialise correctly. Marking task as BROKEN (which will cause other errors)."
+
+576(I) : "Mothership: Initialising supervisor device for application '%s'."

```

Comment:
Note that all of the messages in 576-599 are only used when debugging is enabled.